modules/edit-modes/src/lib/transform-mode.js

```diff
@@ -0,0 +1,319 @@
+// @flow
+
+import envelope from '@turf/envelope';
+import { point, featureCollection, lineString } from '@turf/helpers';
+import { coordEach } from '@turf/meta';
+import { getCoord, getGeom } from '@turf/invariant';
+import turfBuffer from '@turf/buffer';
+import turfCentroid from '@turf/centroid';
+import turfDistance from '@turf/distance';
+import turfBearing from '@turf/bearing';
+import turfTransformTranslate from '@turf/transform-translate';
+import turfTransformScale from '@turf/transform-scale';
+import turfTransformRotate from '@turf/transform-rotate';
+import { getPickedEditHandle } from '../utils';
+import type {
+  PointerMoveEvent,
+  StartDraggingEvent,
+  StopDraggingEvent,
+  DraggingEvent,
+  ModeProps,
+  EditHandleFeature
+} from '../types.js';
+import type { FeatureCollection, Position, Feature } from '../geojson-types.js';
+import { BaseGeoJsonEditMode, getIntermediatePosition } from './geojson-edit-mode';
+import { ImmutableFeatureCollection } from './immutable-feature-collection';
+
+export class TransformMode extends BaseGeoJsonEditMode {
```

Comment:
This mode needs tests as well. Look at draw-rectangle-mode or modify-mode tests as an example.